src/pdb/mod.rs

```diff
@@ -539,8 +555,10 @@ pub enum Row {
     /// Contains the album name, along with an ID of the corresponding artist.
     #[br(pre_assert(page_type == PageType::Tracks))]
     Track {
+        /// Position of start of this row (needed of offset calculations).
+        ///
+        /// **Note:** This is a virtual field and not actually read from the file.
         #[br(temp, parse_with = current_offset)]
-        #[bw(ignore)]
```

Comment:
what does the removal of `#[bw(ignore)]` accomplish? The struct is still writable, its just that the resulting binary is even more "wrong". 
IMO we need to take very special care with these "virtual fields" and make sure they don't end up influencing the reader/writer, otherwise they will not work correctly.

Reply:
> The struct is still writable, its just that the resulting binary is even more "wrong".

It's not though? The enum struct is binread, not binrw.